<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>251</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="assetSearchLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="searchBarLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="searchCustomFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="searchCustomLayout">
          <property name="spacing">
           <number>3</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="assetListViewLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="assetInfoLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="assetInfoViewLayout">
         <property name="spacing">
          <number>3</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="tagViewLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="assetDetailsLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="shotViewLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="assetDependenciesViewLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="keyvalueViewLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="assetCartGroupLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="assetCartLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="assetActionsLayout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
